docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -95,6 +95,25 @@ docker run --rm docker.elastic.co/beats/elastic-agent:{version} elastic-agent co
 
 include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/run-agent-image/widget.asciidoc[]
 
+[TIP] 
+.Running {agent} on a read-only file system
+==== 
+If you'd like to run {agent} in a Docker container on a read-only file 
+system, you can do so by specifying the `--read-only` option.
+{agent} requires a stateful directory to store application data, so
+with the `--read-only` option you also need to use the `--mount` option to
+specify a path to where that data can be stored.
+
+For example:
+
+[source,terminal,subs="attributes"]
+----
+docker run --rm --mount type=tmpfs,destination=/state -e {STATE_PATH}=/state --read-only docker.elastic.co/beats/elastic-agent:8.15.0 <1>
```

Comment:
I would not recommend `type=tmpfs`. It is very much preferred that it be a real mount that can store actually data.

An optional note about the option of using a `tmpfs` could be added, but should have a warning that it can cause data duplicate upon restart of the container (mainly for logs, some metricsets are completely stateless and doesn't cause this issue) as no state information will persist.

Reply:
@blakerouse I've removed `type=tmpfs,` from the command. Let me know if I've got it wrong and instead we should specify a different value for `type`. 

I also added a warning about tmpfs directories not persisting data.